run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+npm install
+npm start
```

Comment:
Consider adding error handling or exit conditions in case npm install fails. This would make the script more robust.